.gitignore

```diff
@@ -25,6 +25,7 @@ Session.vim
 ## Tool
 .valgrindrc
 .cargo
+!/.cargo
```

Comment:
Huh?

Reply:
27 includes `*/.cargo`, 28 negates only top-level config.

I did that to minimize semantic diff, but I can just get rid of 27 as well.

Reply:
Closing as this appears to be a question rather than a request.

Reply:
Does .cargo appear anywhere else? I don't think so, just removing the line seems best.

Reply:
No effect on regressions. I will just get rid of line 27.